aws_lambda_iam_policy_document.tf

```diff
@@ -0,0 +1,35 @@
+data "aws_iam_policy_document" "lambda_config_trust" {
+  count         = "${var.encrypt_ami}"
+  statement {
+    effect = "Allow"
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["lambda.amazonaws.com"]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "lambda_config_policy" {
+  count         = "${var.encrypt_ami}"
+  statement {
+    actions   = ["ec2:Describe*", "ec2:CreateSnapshot", "ec2:CreateImage", "ec2:CreateTags*", "ec2:CopyImage"]
```

Comment:
Where is the `effect = "Allow"` statement? is that not required anymore? this is still TF 0.11 right?

Reply:
missed that, added it. Still worked without it though